koku/api/report/queries.py

```diff
@@ -115,6 +117,13 @@ def query_table_filter_keys(self):
         filter_keys = set(self.parameters.get("filter", {}).keys())
         return filter_keys.difference(excluded_filters)
 
+    @cached_property
+    def query_table_exclude_keys(self):
+        """Return the filter keys specific for selecting the query table."""
+        excluded_filters = {"time_scope_value", "time_scope_units", "resolution", "limit", "offset"}
+        filter_keys = set(self.parameters.get("exclude", {}).keys())
+        return filter_keys.difference(excluded_filters)
```

Comment:
Not sure I get this block of code. The serializer prevents the `excluded_filters` items. Feels like this could just be:
```suggestion
    def query_table_exclude_keys(self):
        """Return the exclude keys specific for selecting the query table."""
        return set(self.parameters.get("exclude", {}).keys())
```